### ISSUE TITLE
关于PNNconcat的问题。

### ISSUE THREAD

**Issue by Greak-1124** [ZiyaoGeng/Recommender-System-with-TF2.0#74]
你好，请问在PNN中，的`l_1 = tf.nn.relu(tf.concat([l_z + l_p + self.l_b], axis=-1))`是什么意思呢？l_z 和l_p分别是对输入进行线性变换和对内积/外积进行了变换，再融合。但是self.l_b是定义的一个权重，为什么可以直接将它们相加呢？相加以后再concat又是什么意思？希望能解答一下，谢谢！

**Comment by ZiyaoGeng** [comment]
参考原文
